kivy/core/audio_output/audio_sdl3.pyx

```diff
@@ -59,7 +59,7 @@ cdef mix_init():
 
     # avoid next call
     if mix_is_init != 0:
-        return
+        return mix_is_init > 0 ? 1 : 0;
```

Comment:
This is not a valid Cython syntax unfortunately ☺️

Reply:
Hehe fixed it now.